import-scripts/annotation_suite/standardize_mutation_data.py

```diff
@@ -0,0 +1,1391 @@
+MATCHED_NORMAL_SEQ_ALLELE1_COLUMNS = ["Match_Norm_Seq_Allele1", "mutated_to_allele"]
+MATCHED_NORMAL_SEQ_ALLELE2_COLUMNS = ["Match_Norm_Seq_Allele2", "control_genotype"]
+TUMOR_SAMPLE_BARCODE_COLUMNS = ["Tumor_Sample_Barcode", "analyzed_sample_id"]
+
+# VCF KEYS FOR RESOLVING WHICH VCF PIPIELINE WAS USED
+VCF_STRELKA_KEY_COLUMNS = ["AU", "CU", "GU", "TU"]
+VCF_CAVEMAN_KEY_COLUMNS = ["FAZ", "FCZ", "FGZ", "FTZ", "RAZ", "RCZ", "RGZ", "RTZ"]
+VCF_ION_TORRENT_KEY_COLUMNS = ["AO", "RO"]
+VCF_DELLY_KEY_COLUMNS = ["RR", "RV"]
+VCF_CGPPINDEL_KEY_COLUMNS = ["PP", "NP", "PR", "NR"]
+VCF_ALT_ALLELE_FRACTION_KEY_COLUMNS = ["FA", "DP"]
+VCF_MPILEUP_BCFTOOLS_KEY_COLUMNS = ["DV", "DP"]
+
+VCF_REFERENCE_ALLELE_VALUE = "0"
+VCF_VARIANT_ALLELE1_VALUE = "1"
+VCF_VARIANT_ALLELE2_VALUE = "2"
+VCF_DEFAULT_VARIANT_ALLELE_IDX = 1
+
+# ----------------------------------------------------------------
+
+def get_file_header(filename):
+    """ Returns the file header as a list. """
+    header = []
+    with open(filename, "rU") as data_file:
+        for line in data_file.readlines():
+            if line.startswith("#"):
+                continue
+            header = list(map(str.strip, line.split("\t")))
+            break
+    return header
+
+def process_datum(value):
+    """
+        Returns a cleaned up data value.
+        A null or 'empty' value is always returned as an empty string.
+    """
+    try:
+        vfixed = value.strip()
+    except AttributeError:
+        vfixed = ""
+    if vfixed in ["", "", "N/A", None, ".", "?"]:
+        return ""
+    return vfixed.replace('"',"")
+
+def resolve_tumor_seq_allele(data, ref_allele):
+    """ Resolve the tumor seq allele. """
+    tum_seq_allele1 = ""
+    tum_seq_allele2 = ""
+
+    if MUTATED_TO_ALLELE_COLUMN in data.keys():
+        return data[MUTATED_TO_ALLELE_COLUMN]
+
+    for i,column in enumerate(TUMOR_SEQ_ALLELE1_COLUMNS):
+        if column in data.keys():
+            tum_seq_allele1 = process_datum(data[column])
+            tum_seq_allele2 = process_datum(data[TUMOR_SEQ_ALLELE2_COLUMNS[i]])
+
+            # if at least one is not empty then exit for-loop
+            if tum_seq_allele1 != "" or tum_seq_allele2 != "":
+                break
+
+    # if temp tum seq alleles are not equal then set tum_seq_allele that doesn't equal ref_allele
+    # otherwise there is probably something wrong with the data
+    if tum_seq_allele1 == "" and tum_seq_allele2 == "":
+        return ""
+    if tum_seq_allele1 == ref_allele:
+        return tum_seq_allele2
+    else:
+        return tum_seq_allele1
+
+
+def print_warning(message):
+    print(message, ERROR_FILE)
+
+def resolve_variant_type(data, ref_allele, tum_seq_allele):
+    """ Resolve variant type.  """
+    for column in VARIANT_TYPE_COLUMNS:
+        if column in data.keys():
+            variant_type = process_datum(data[column].upper())
+            break
+    if variant_type == "1":
```

Comment:
are integers something we have seen before in the variant_type column in MAFs from genie?

Reply:
Same comment as before - this script was written for handling more than just what we expect from GENIE (or Tom expects from the individual centers)